src/views/catalog/CreateFromInstanceTypes/components/AddBootableVolumeModal/utils/utils.ts

```diff
@@ -52,7 +52,7 @@ export const createDataSource =
     const bootableVolumeToCreate = produce(emptySourceDataVolume, (draftBootableVolume) => {
       draftBootableVolume.metadata.name = bootableVolumeName;
       draftBootableVolume.spec.storage.resources.requests.storage = size;
-      if (storageClassName && isUploadForm) {
+      if (storageClassName) {
```

Comment:
u sure this change dont break anything else? as now it will set the SC for everything, is this the right act we want?

Reply:
yes